src/libraries/Microsoft.PowerFx.Interpreter/Marshal/TypeMarshallerCache.cs

```diff
@@ -39,20 +39,25 @@ public class TypeMarshallerCache
         /// </summary>
         private readonly IEnumerable<ITypeMarshallerProvider> _marshallers;
 
+        private readonly IEnumerable<IDynamicTypeMarshaller> _dynamicMarshallers;
+
         // Take a private array to get a snapshot and ensure the enumeration doesn't change
-        private TypeMarshallerCache(ITypeMarshallerProvider[] marshallers)
+        private TypeMarshallerCache(
+            IEnumerable<ITypeMarshallerProvider> marshallers,
+            IDynamicTypeMarshaller[] dynamicMarshallers = null)
         {
             _marshallers = marshallers;
+            _dynamicMarshallers = dynamicMarshallers;
         }
 
         /// <summary>
         /// Initializes a new instance of the <see cref="TypeMarshallerCache"/> class.
         /// Create marshaller with default list.
         /// </summary>
         public TypeMarshallerCache()
-            : this(_defaults.ToArray())
+            : this(_defaults.ToArray(), null)
```

Comment:
Why is null being passed in here?

Reply:
because by default, we don't have any IDynamicTypeMarshaller